doc/source/ray-core/handling-dependencies.rst

```diff
@@ -373,6 +373,22 @@ file and use `uv run --with-requirements requirements.txt` for your `py_executab
 or use the `--with` flag to specify individual requirements.
 
 
+.. tip::
+
+  In order to make this functionality available in a convenient way without having
+  to specify `py_executable` in the runtime environment, you can use the following
+  runtime environment hook:
+
+  .. code-block:: sh
+
+    export RAY_RUNTIME_ENV_HOOK=ray._private.runtime_env.uv_runtime_env_hook.hook
+
+  It will set the `py_executable` to `uv run` with the same parameters as the driver
+  and will also set the `working_dir` to the same working directory that the driver
+  is using (either the current working directory or the `--directory` in `uv run`).
+  In the future we might make this behavior the default.
```

Comment:
This assumes that I used `uv run` to run the driver process right? I don't think I'd understand it from this note alone. Maybe add an explicit code sample like:
```suggestion
    export RAY_RUNTIME_ENV_HOOK=ray._private.runtime_env.uv_runtime_env_hook.hook
    uv run my_script.py

  It will set the `py_executable` to `uv run` with the same parameters as the driver
  and will also set the `working_dir` to the same working directory that the driver
  is using (either the current working directory or the `--directory` in `uv run`).
  In the future we might make this behavior the default.
```